common/README.md

```diff
@@ -4,54 +4,43 @@ This directory manages GPII infrastructure in [Google Cloud Project (GCP)](https
-1. There's no automation for destroying the Project and starting over. I usually use the GCP Dashboard.
-   * Note that "deleting" a Project really marks it for deletion in 30 days. You can't create a new Project with the same name until the old one is culled.
-
-## One-time Google Cloud Account Setup
-* https://cloud.google.com/resource-manager/docs/quickstart-organizations
-   * G Suite: "The first time a user in your domain creates a project or billing account, the Organization resource is automatically created and linked to your company’s G Suite account. The current project and all future projects will automatically belong to the organization."
-      * @mrtyler believes he did this when he created his Free Trial account using his RtF email address.
-   * "If you're the Super Admin of your G Suite domain account, you can add yourself and others as the Organization Admin of the corresponding Organization. For instructions on adding Organization Admins, see Adding an organization admin."
-* https://cloud.google.com/resource-manager/docs/creating-managing-organization#adding_an_organization_admin
-   * Manually create IAMs for Ops Team. Assign role "Organization Policy Administrator" and "Billing Account User".
-      * Actually this isn't working either -- even with admin privileges, Alfredo can't attach his Project to the Official Billing Account. Alfredo is investigating.
-* https://cloud.google.com/resource-manager/docs/quickstart-organizations#create_a_billing_account
-   * Manually create IAM for Eugene. Assign role "Billing Account Administrator".
-   * Eugene creates Billing Account, "Official". Fills in contact info, payment info.
-      * This Billing Account didn't show up for me until Eugene added me as billing admin for that billing account (even when I already had Organization Policy Administrator and Billing Account Administrator).
-      * Giving myself Billing Account Administrator did allow me to see the Billing Account sgithens created for his Free Trial Account.
-      * In spite of all those privileges, I can't Manage Payment Users for any Billing Accounts other than the one from my Free Trial Account.
-   * Send billing emails to accounts-payable@rtf-us.org -- https://cloud.google.com/billing/docs/how-to/modify-contacts
-      * Billing -> Official -> Payment Settings -> Payments Users -> Manage Payments Users -> Add a New User. Leave all Permissions unchecked. Leave Primary Contact unchecked (there can be only one, and it's Eugene). Confirm invitation email.
+1. By default you'll use the RtF Organization and Billing Account.
+   * You can use a different Organization or Billing Account, e.g. from a GCP Free Trial Account, with `export ORGANIZATION_ID=111111111111` and/or `export BILLING_ID=222222-222222-222222`.
+1. Check that [you have the AWS credentials](../aws#configure-cloud-provider-credentials).
+   * Copy the `credentials` file to the `common/.config/prd/aws` directory.
```

Comment:
I think this is not correct anymore, looks like the new volume code always expect the credentials to be at `~/.aws` - this seems inconsistent with the rest?